geo.txt

```diff
@@ -1681,15 +1705,14 @@ NAME = C3Pd
 # Source paper: Bittner DM, Zaleski DP, Tew DP, Walker NR, Legon AC (2016) Highly unsaturated platinum and palladium
 carbenes PtC3 and PdC3 isolated and characterized in the gas phase. Angew Chem 128(11):3832-3835; Angew
 Chem Int Ed Engl 55(11):3768-3771
+# C(1) = C(2) 1.3009 and C(2) = C(3) 1.2789 fixed to the value calculated at the level CCSD(T)_ae/aug-cc-pwCV5Z-PP(for Pd) and corrected for r0 ‒ re
+difference assumed to that for PtC3. 
```

Comment:
It seems that the line wrapped a bit too early?

Reply:
Think ill need to make a new line.